pydeconz/models/sensor/presence.py

```diff
@@ -85,6 +85,15 @@ class PresenceConfigTriggerDistance(enum.StrEnum):
     FAR = "far"
     MEDIUM = "medium"
     NEAR = "near"
+    ONE = "1"                                                     
```

Comment:
Could you just add a comment to specify that this is in meters?
```suggestion

    # Trigger distance in meters
    ONE = "1"                                                     
```